docs/parts/monorepo-support.md

```diff
@@ -0,0 +1,70 @@
+You can also see demos for each of the package managers in this [monorepo demo collection](https://stackblitz.com/@d3lm/collections/workspaces).
+
+### npm
+
+Your project repository may look like this:
+
+```
+workspace-project
+├─ packages
+│  ├─ backend
+│  │  ├─ package.json
+│  │  └─ index.js
+│  ├─ frontend
+│  │  ├─ package.json
+│  │  └─ index.js
+│  └─ common
+│     ├─ package.json
+│     └─ index.js
+└─ package.json
+```
+
+To define a workspace, add the `workspaces` field to the `package.json`:
+
+```
+"workspaces": [
+  "packages/*"
+],
+```
+
+This refers to every sub-directory inside `packages` which contains a `package.json`.
+
+Check [npm documentation](https://docs.npmjs.com/cli/v7/using-npm/workspaces) for more information or [this npm-based monorepo demo](https://stackblitz.com/edit/node-4cygsf?file=README.md) for reference. 
```

Comment:
I think “npm documentation” might be incorrect, and correct forms would be “the npm documentation” or “npm’s documentation”?

Reply:
```suggestion
Check the [npm documentation](https://docs.npmjs.com/cli/v7/using-npm/workspaces) for more information or [this npm-based monorepo demo](https://stackblitz.com/edit/node-4cygsf?file=README.md) for reference. 
```